translate/tests/system/test_vpcsc.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for VPC-SC."""
+
+import os
+import pytest
+
+from google.api_core import exceptions
+from google.cloud import translate_v3beta1
+from google.cloud.translate_v3beta1.proto import translation_service_pb2
+from google.longrunning import operations_pb2
+
+PROJECT_INSIDE = os.environ.get("PROJECT_ID", None)
+PROJECT_OUTSIDE = os.environ.get(
+    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT", None
+)
+IS_INSIDE_VPCSC = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "true")
+
+class TestVPCServiceControl(object):
+    @classmethod
+    def setup(self):
+        self._client = translate_v3beta1.TranslationServiceClient()
+        self._parent_inside = self._client.location_path(PROJECT_INSIDE, 'us-central1')
+        self._parent_outside = self._client.location_path(PROJECT_OUTSIDE, 'us-central1')
+
+        def make_glossary_name(project_id):
+          return 'projects/{0}/locations/us-central1/glossaries/fake_glossary'.format(project_id)
+        self._glossary_name_inside = make_glossary_name(PROJECT_INSIDE)
+        self._glossary_name_outside = make_glossary_name(PROJECT_OUTSIDE)
+
+    @staticmethod
+    def _is_rejected(call):
+        try:
+            responses = call()
+            print("responses: ", responses)
+        except exceptions.PermissionDenied as e:
+            print("PermissionDenied Exception: ", e)
+            return e.message == "Request is prohibited by organization's policy"
+        except Exception as e:
+            print("Other Exception: ", e)
+            pass
+        return False
+
+    @staticmethod
+    def _do_test(delayed_inside, delayed_outside):
+        if IS_INSIDE_VPCSC.lower() == "true":
+            assert TestVPCServiceControl._is_rejected(delayed_outside)
+            assert not (TestVPCServiceControl._is_rejected(delayed_inside))
+        else:
+            assert not (TestVPCServiceControl._is_rejected(delayed_outside))
+            assert TestVPCServiceControl._is_rejected(delayed_inside)
+
+    @pytest.mark.skipif(
+        PROJECT_INSIDE is None, reason="Missing environment variable: PROJECT_ID"
```

Comment:
It is possible the environment has a `PROJECT_ID` set even if there isn't a vpc. 

Would it make sense to trigger this on `IS_INSIDE_VPC`?

Reply:
Yes that makes more sense. Changed to using IS_INSIDE_VPC